.github/workflows/create-charts.sh

```diff
@@ -65,26 +65,39 @@ update_platform_services_charts() {
         echo '  - name: hypertrace-graphql-service'
         echo '    repository:' $HELM_GCS_REPO
         echo '    version:' $hypertrace_graphql_version
-        echo '  - name: attribute-service'
-        echo '    repository:' $HELM_GCS_REPO
-        echo '    version:' $attribute_service_version
         echo '  - name: gateway-service'
         echo '    repository:' $HELM_GCS_REPO
         echo '    version:' $gateway_service_version
+        echo '    condition: merge-query-services.disabled'
         echo '  - name: query-service'
         echo '    repository:' $HELM_GCS_REPO
         echo '    version:' $query_service_version
+        echo '    condition: merge-query-services.disabled'
         echo '  - name: entity-service'
         echo '    repository:' $HELM_GCS_REPO
         echo '    version:' $entity_service_version
+        echo '    condition: merge-query-services.disabled'
+        echo '  - name: attribute-service'
+        echo '    repository:' $HELM_GCS_REPO
+        echo '    version:' $attribute_service_version
+        echo '    condition: merge-query-services.disabled'
```

Comment:
@rish691, I can see a bit confusion here. The enable or disable here won't decide if service is enabled or not. You have set a value true or false for enabled variable in `values.yaml`. We are just setting conditional variable here.

Look at the postgres as a example here: https://github.com/hypertrace/hypertrace/blob/main/kubernetes/data-services/values.yaml#L108-L109

I will suggest to stick to `enabled` for variable name and then we can set values for that.

Reply:
Ofcourse it is the value of the variable in the conditional which will determine if the dependency is added or not. 
So name of the variable is immaterial. 

Now, coming to the change, helm doesn't allow for negation of variables, so you can't do something like 
`condition: !merge-query-services.enabled` or anything similar. Which implies that you can't depend on a single var
to achieve switching between charts based on condition. 

Currently it is being controlled by toggling value of vars in `merge-query-services.enabled` and `merge-query-services.disabled`

  
```
merge-query-services: 
  enabled: false
  disabled: true
```

Reply:
ohh. I see it now. You need 2 variables. got it